libra-node/src/main_node.rs

```diff
@@ -52,10 +54,16 @@ impl Drop for LibraHandle {
 }
 
 fn setup_executor(config: &NodeConfig) -> Arc<Mutex<Executor<LibraVM>>> {
+    let rt = Executor::<LibraVM>::create_runtime();
```

Comment:
I'm not sure I like this `create_runtime` API. First we call `Executor::create_runtime` to construct a runtime, then we pass the `rt` back to `Executor::new` to construct an executor. Do you think it might be a bit better to construct the runtime outside executor?

Reply:
Just to make code shorter on the call sites. Will be removed pretty soon. :D